index.html

```diff
@@ -437,8 +456,29 @@ <h6>Importance</h6>
 
 						---
 
```

Comment:
Since the introduction of div and span included block/inline and the idea of semantic meaning, it would make sense to introduce those concepts formally.